project/database/migration.py

```
@@ -0,0 +1,232 @@
+            verification["json_warnings"] = json_count
+
+            # Count database warnings
+            service = get_warning_service()
+            try:
+                # This is a simplified count - in reality you'd need to query by guild
+                # For now, we'll just check if we have any warnings
+                from .models import SecureWarning
+
+                db_count = (
+                    service.db.query(SecureWarning)
+                    .filter(SecureWarning.is_deleted == False)
```

Ruff (0.12.2)

Avoid equality comparisons to `False`; use `not SecureWarning.is_deleted:` for false checks

Replace with `not SecureWarning.is_deleted`

(E712)